src/_base/_twig/docker-compose.yml/service/mongodb.yml.twig

```diff
@@ -0,0 +1,15 @@
+  mongodb:
+    image: {{ @('services.mongodb.image') }}
+    environment: {{ to_nice_yaml(deep_merge([
+        @('services.mongodb.environment'),
+        @('services.mongodb.environment_secrets')
+      ]), 2, 6) | raw }}
+    labels:
+      - traefik.enable=false
+    networks:
+      - private
+    restart: always
```

Comment:
Is this intentional? Currently we only have that on workspace provided global services - traefik/proxy and mail

Reply:
I can't remember